pkg/utils/downloaders.go

```diff
@@ -64,7 +66,7 @@ func (desc UEFIDescription) image(latest bool) (string, error) {
 }
 
 //DownloadEveLive pulls EVE live image from docker
-func DownloadEveLive(eve EVEDescription, uefi UEFIDescription, outputFile string) (err error) {
+func DownloadEveLive(model models.DevModel, eve EVEDescription, uefi UEFIDescription, outputFile string) (err error) {
```

Comment:
Can we add string with model name into EVEDescription type instead of adding new argument?

Reply:
Yes, I also like this option more, but there was a meeting, and it seemed to me that it was better not to do it. Perhaps during the discussion I was misunderstood.